### PR TITLE
Change NovelCoronaAPIv1 to CovidAPIv1

### PR DESCRIPTION
**Pull request by melsaa**
### Related Issue:

Fix issue #59 

### Proposed Changes

* Change NovelCoronaAPIv1 to CovidAPIv1
* Skip obsolete tests

### Additonal Info

- any additional information or context

### Checklist

* [x] Tests
* [ ] Translations
* [ ] Documentation

### Screenshots

* screenshot 1

### CODE REVIEW

File: app/routers/v1/router_api_v1.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def reload_model_api_v1(func):
     @wraps(func)
     def wrapper(*args, **kwargs):
         global novel_corona_api, dt, ts
```
Comment (nat236919) [inline app/routers/v1/router_api_v1.py:23]: Would you mind changing **novel_corona_api** to **COVID_API_V1** in this file as well?

Comment (melsaa) [reply app/routers/v1/router_api_v1.py]: done

**Review by nat236919 — APPROVED** [review]
LGTM

**Comment by nat236919** [comment]
@all-contributors please add @melsaa for code

**Pull request merged**